src/editor/contentState/selectionCtrl.js

```diff
@@ -0,0 +1,43 @@
+const selectionCtrl = ContentState => {
+  ContentState.prototype.tableCellHandler = function (event) {
+    const { start, end } = this.cursor
+    const startBlock = this.getBlock(start.key)
+    const endBlock = this.getBlock(end.key)
+    const { type: startType } = startBlock
+    const { type: endType } = endBlock
+    if (/th|td/.test(startType) && /th|td/.test(endType)) {
+      event.preventDefault()
+      event.stopPropagation()
+      if (start.key === end.key) {
+        const { text, key } = startBlock
+        this.cursor = {
+          start: { key, offset: 0 },
+          end: { key, offset: text.length }
+        }
+      } else {
+        // Get table from start block
+        //   table -> thead or tbody -> tr -> th or td (cell)
+        const table = this.getParent(this.getParent(this.getParent(startBlock)))
+        if (!table || table.type !== 'table') {
+          console.error('No table found or invalid type.')
+          return
+        }
+        // Get last table block
+        const lastTableBlock = this.getLastChild(this.getLastChild(this.getLastChild(table)))
+        if (!lastTableBlock || !/th|td/.test(lastTableBlock.type)) {
+          console.error('No table cell found or invalid type.')
+          return
+        }
+        const { key: startKey } = table
```

Comment:
table block is has no text, I think the first cell in the table need to have the start cursor.

Reply:
Mhh... Its works without searching for the first cell. I think table cursor and first cells cursor are the same because the cursor is set to the first text in the table.

Reply:
@fxha I have a question not about the codes, but the feature of selecting one table.

If we select one table, can we copy and paste it to other place?

Reply:
No, it's just a string `a b c d e f`.

| a    | b    | c    |
| ---- | ---- | ---- |
| d    | e    | f    |

Do you want to copy a markdown table?

Reply:
What I want is when we select all the contents of one table, we can copy and paste the table. not only the text. What's you opinions?

Reply:
So we copy the markdown table? E.g.

```
| a    | b    | c    |
| ---- | ---- | ---- |
| d    | e    | f    |
```

Reply:
I think `table copy and paste` will be another issue or PR, let's fix this first.